dev/com.ibm.ws.classloading/src/com/ibm/ws/classloading/internal/AppClassLoader.java

```diff
@@ -264,6 +264,18 @@ public URL getResource(String name) {
         return result;
     }
 
+    @Override
+    public InputStream getResourceAsStream(String name) {
+        InputStream result = findStreamCommonLibraryClassLoaders(name, beforeApp);
```

Comment:
does the ThreadIdentityManager logic need to be here like it is in findResourceInternal?